acceptance/openstack/ecs/v1/cloudservers_test.go

```diff
@@ -12,11 +12,14 @@ func TestCloudServerLifecycle(t *testing.T) {
 	client, err := clients.NewComputeV1Client()
 	th.AssertNoErr(t, err)
 
+	// Get ECSv1 createOpts
 	createOpts := getCloudServerCreateOpts(t)
 	createOpts.DryRun = true
 
+	// Check ECSv1 createOpts
 	err = dryRunCloudServerConfig(t, client, createOpts)
 	th.AssertNoErr(t, err)
+	t.Logf("CreateOpts are true for creating a cloudServer")
```

Comment:
It's better to say that `CreateOpts are ok`, than `true` :) Or even say that create options validation passes